chain/blocksync/blocksync_client.go

```diff
@@ -284,25 +285,25 @@ func (bs *BlockSync) fetchBlocksBlockSync(ctx context.Context, p peer.ID, req *B
 	ctx, span := trace.StartSpan(ctx, "blockSyncFetch")
 	defer span.End()
 
-	start := time.Now()
+	start := build.Clock.Now()
 	s, err := bs.host.NewStream(inet.WithNoDial(ctx, "should already have connection"), p, BlockSyncProtocolID)
 	if err != nil {
 		bs.RemovePeer(p)
 		return nil, xerrors.Errorf("failed to open stream to peer: %w", err)
 	}
-	_ = s.SetWriteDeadline(time.Now().Add(5 * time.Second))
+	_ = s.SetWriteDeadline(build.Clock.Now().Add(5 * time.Second))
```

Comment:
Same with deadline here.

Reply:
Fixed in [`fbf6596` (#2362)](https://github.com/filecoin-project/lotus/pull/2362/commits/fbf6596f20744628212d4b6a5c92ded32ae0693f).